paasta_tools/cli/cmds/spark_run.py

```diff
@@ -1148,6 +1166,12 @@ def update_args_from_tronfig(args: argparse.Namespace) -> Optional[Dict[str, str
 
 
 def paasta_spark_run(args: argparse.Namespace) -> int:
+    if args.get_eks_token_via_iam_user and os.getuid() != 0:
+        print("Re-run proccess as root", file=sys.stderr)
```

Comment:
Nit: update text to match other places in paasta, ie "Re-executing paasta spark-run with sudo.."